### DISCUSSION TITLE
Repeatable subtract vs. subtraction of union

### DISCUSSION THREAD

**Discussion by eevleevs** [jscad/OpenJSCAD.org#1313] [General]
Hello, I noticed that `subtract(a, ...b)` is potentially much slower than `subtract(a, union(b))` when b has many items. 
Why not always perform a union of the items to be subtracted? Are there cases where the operations are not equivalent?

**Comment by z3dev** [comment]
Hi @eevleevs 

Nice observation! We are always looking for ways to improve the performance of the operations. This is definitely going to be investigated.
